Assets/Character/AaronTestAssets/PlayerMovement.cs

```diff
@@ -100,16 +105,16 @@ public class PlayerMovement : MonoBehaviour {
     }
 
     //use raycasting to determine OnGround
-    //jump "fails" if pressed during window where you're not touching the ground yet but already "OnGround", but input queue should fix this
-    void checkBottom(string check)
+    //jump "fails" if pressed during the window when you're not touching the ground yet but already "OnGround",
+    //but input queue should fix this
+    void checkBottom()
```

Comment:
is this checking if we're touching the ground? Make intention clear with function name and capitalize first letter